docs/README.md

```diff
@@ -1,10 +1,59 @@
 [Return to Project Root](../)
 
 # User Documentation
-tbd
+sail-operator manages the lifecycle of your Istio control planes and aims to become a vehicle for automated integration of Istio with compatible third-party software. Instead of creating a new configuration schema, sail-operator APIs are built around Istio's helm chart APIs. All installation and configuration options that are exposed by Istio's helm charts are available through the sail-operator CRDs' `values` fields.
```

Comment:
> aims to become a vehicle for automated integration of Istio with compatible third-party software

I'm not really sure what this means.

Reply:
yeah I'm removing that. it's a bit too vague anyway and we don't actually do that right now